org-node/src/client.rs

```diff
@@ -194,7 +195,7 @@ impl Client {
     }
 
     /// Run the client. This function runs indefinitely until a fatal error occurs.
-    pub async fn run(self, rt: tokio::runtime::Handle) {
+    pub async fn run(self, rt: tokio::runtime::Handle, _ws_tx: UnboundedSender<WsEvent>) {
```

Comment:
Not using this handler, but could envision the client wanting to have a handle to the websocket clients.

Reply:
Used now by gossip protocol events.